$NetBSD: patch-ad,v 1.1 2011/01/21 13:09:51 dmcmahill Exp $

Remove level 0 markup which is not allowed for an 'article' document.

--- Documentation/tutorial.txt.orig	2011-01-21 12:56:56.000000000 +0000
+++ Documentation/tutorial.txt	2011-01-21 12:57:06.000000000 +0000
@@ -1,4 +1,4 @@
 StGIT Tutorial
-##############
+===============
 
 
